docs/manual/article-management/insert-tags.en.md

```diff
@@ -18,7 +18,7 @@ With these insert tags you can create links to other pages or articles. You only
 | Insert tag | Description                                                                                                                                                                                                       |
 | ---------- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `{{link::*}}` | This tag is replaced with HTML code for a link. The parameter can be the ID or alias of an internal page or an absolute URL. <sup>[[1]](#parameter-absolute)</sup><sup>,</sup> <sup>[[2]](#parameter-blank)</sup> |
-| `{{link::back}}` | This tag is replaced with a link to the last page visited. Can also be used as `{{link_open::back}}`, `{{link_url::back}}`and `{{link_title::back}}`.                                                             |
+| `{{link::back}}` | This tag is replaced with a link to the last page visited. Can also be used as `{{link_open::back}}`, `{{link_url::back}}`and `{{link_title::back}}`. Note: Deprecated since Contao 4.13, not available anymore in **Contao 5.0**).                                                        |
```

Comment:
```suggestion
| `{{link::back}}` | This tag is replaced with a link to the last page visited. Can also be used as `{{link_open::back}}`, `{{link_url::back}}`and `{{link_title::back}}` (deprecated since **Contao 4.13**, not available anymore in **Contao 5.0**).                                                        |
```

That's the format we used previously.